app/dashboard/embed.py

```diff
@@ -218,9 +219,10 @@ def embed(request):
             i += 1
             value_eth = str(round(bounty.value_in_eth/10**18, 2)) + "ETH" if bounty.value_in_eth else ""
             value_in_usdt = str(round(bounty.value_in_usdt, 2)) + "USD" if bounty.value_in_usdt else ""
+            eth_value_in_usdt = str(round(get_eth_to_usdt(), 2)) + "/ETH"
```

Comment:
what if the bounty is denomated in some other token besides ETH?  

in this case, the message we'd want the backend to produce is the conversion rate for MKR/USD, a la`$x.xx MKR/USD`

Reply:
I was thinking about this but just included ETH since it appears the front end is hardwired to ETH. 

So to clarify, this should be the USD/{TOKEN} right? It should always return the current USDT rate of the token correct? In other words, if it token is MKR the string to should look something like:

```
"(201.0 USD @ 700 USD/MKR)"
... or ...
"(201.0 USD @ $700/MKR)"
depending on the answer to my question above
```
Also, should the currency also be dynamic or always asumed to be USDT?

Reply:
Sounds good. Will work on these changes this morning.